code/modules/mapfluff/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -271,6 +276,25 @@
 		deconstruct(TRUE)
 		return TRUE
 
+/obj/structure/closet/crate/grave/container_resist_act(mob/living/user)
+	if(opened)
+		return
+	// The player is trying to dig themselves out of an early grave
+	user.changeNext_move(CLICK_CD_BREAKOUT)
+	user.last_special = world.time + CLICK_CD_BREAKOUT
+	user.visible_message(span_warning("[src]'s dirt begins to shift and rumble!"), \
+		span_notice("You desperately begin to claw at the dirt around you, trying to force yourself upwards through the soil... (this will take about [DisplayTimeText(breakout_time)].)"), \
+		span_hear("You hear the sound of shifting dirt from [src]."))
```

Comment:
```suggestion
	user.visible_message(
		span_warning("[src]'s dirt begins to shift and rumble!"),
		span_notice("You desperately begin to claw at the dirt around you, trying to force yourself upwards through the soil... (this will take about [DisplayTimeText(breakout_time)].)"),
		span_hear("You hear the sound of shifting dirt from [src]."),
	)
```

this is the way to multiline for modern DM